Source/Core/DolphinQt2/Settings.h

```diff
@@ -131,6 +132,7 @@ class Settings final : public QObject
 signals:
   void ConfigChanged();
   void EmulationStateChanged(Core::State new_state);
+  void UpdateDisasmDialog();
```

Comment:
I don't think this belongs in Settings.

Also, the naming should be more consistent (Disasm or Disassembler, pick one).